examples/vertx-opentracing/README.md

```diff
@@ -12,58 +12,64 @@ Manager is performed via publishing a message on a shared destination.
 
 The other operation that can be performed by an end user is to list the current orders associated with an account id.
 
-== Run
+## Run
 
-First step is to start up the APM server. Then open a console window from this folder and run:
+First step is to start up the APM server. Then open a console window from this folder and either run the command directly using maven:
 
-[source,shell]
-----
+```shell
 $ export HAWKULAR_APM_URI=http://localhost:8080
 $ export HAWKULAR_APM_USERNAME=...
 $ export HAWKULAR_APM_PASSWORD=...
 $ mvn clean install
 $ mvn exec:java
-----
+```
 
-== Example requests
+or use docker:
+
+```shell
+$ docker build -t hawkular/apm-example-vertx-opentracing .
```

Comment:
This won't be necessary. Image will be available on docker hub. Better would be to provide `docker pull` command to get the latest image.